console_backend/src/ntrip_tab.rs

```diff
@@ -284,8 +285,11 @@ fn main(
         }
     })?;
 
-    transfer.borrow_mut().write_function(|data| {
-        if let Err(e) = msg_sender.write_all(data) {
+    let (rtcm_tx, rtcm_rx) = channel::unbounded::<Vec<u8>>();
+    let rtcm2sbp = RtcmConverter::new(rtcm_rx);
+    rtcm2sbp.start(msg_sender);
+    transfer.borrow_mut().write_function(move |data| {
+        if let Err(e) = rtcm_tx.send(data.to_owned()) {
```

Comment:
send rtcm3tosbp output back directly to msg_sender

Reply:
still doesn't seem to be doing anything... maybe `MsgSender` is incorrect place to send?

Reply:
resolved, this seems to be a device issue. works fine sending SBP back.